sgkit/io/plink/plink_writer.py

```diff
@@ -20,6 +31,20 @@ def write_plink(
 ) -> None:
```

Comment:
Should this have a `missing: Hashable ='.' ` parameter? To specify the missing parent value and default to the new value from `read_plink`.

Reply:
I'm not sure this is needed. We don't parameterise the missing value in VCF for example, so I think it's ok to have the sgkit default of "." used here too.

Reply:
Ah, I see it's handled by `values = np.where(values == STR_MISSING, "0", values)`. 
> We don't parameterise the missing value in VCF

Good point, but "." is the missing value in the VCF spec where as "0" is the missing value in a plink fam file. Although, a "0" will stay as a "0" so I guess both cases are covered anyway 🤷

> the sgkit default of "."

We should note that in the main documentation (not needed in this PR)